### ISSUE TITLE
Battlepass broken on (my?) mac

### ISSUE THREAD

**Issue by reutsharabani** [ValveSoftware/Dota-2#2117]
#### Your system information

* System information from steam (`Steam` -> `Help` -> `System Information`) in a [gist](https://gist.github.com/): 
https://gist.github.com/reutsharabani/b87192e87526ec68bbbc7f5688a6792a

* Have you checked for system updates?: Yes
* Are you using the latest stable video driver available for your system? Yes
* Have you verified the game files?: Yes

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large pastes as a [Github Gist](https://gist.github.com/).

Battle pass does not start on mac (again...)

These are my logs:

https://gist.github.com/reutsharabani/33f0863aa2e966fdef1161c5b96937d5

I ran:

```sh
exec ~/Library/Application\ Support/Steam/steamapps/common/dota\ 2\ beta/game/dota.sh
```

#### Steps for reproducing this issue:

1. buy mac
2. run dota

**Comment by alfer-aws** [comment]
+1

**Comment by viciecal** [comment]
+1 same happening to me - Was running fine yesterday.

**Comment by WillianJunior** [comment]
+1 Same here on Ubuntu.
Game opens normally, plays normally, but breaks as soon as I click on anything from battle pass.

**Comment by alfer-aws** [comment]
Same logs as well:

`Fontconfig warning: "/Users/alfer/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/core/panorama/fonts/conf.d/41-repl-os-win.conf", line 148: Having multiple values in <test> isn't supported and may not work as expected
Fontconfig warning: "/Users/alfer/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/core/panorama/fonts/conf.d/41-repl-os-win.conf", line 160: Having multiple values in <test> isn't supported and may not work as expected
Loaded /Users/alfer/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libcairo.dylib, got 0x0
 failed to dlopen "/Users/alfer/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libcairo.dylib" error=dlopen(/Users/alfer/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libcairo.dylib, 0x0009): tried: '/Users/alfer/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/libcairo.dylib' (no such file), '/libcairo.dylib' (no such file), '/Users/alfer/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libcairo.dylib' (no such file)
Loaded libcairo.dylib, got 0x0
 failed to dlopen "libcairo.dylib" error=dlopen(libcairo.dylib, 0x0009): tried: '/Users/alfer/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/libcairo.dylib' (no such file), '/libcairo.dylib' (no such file), '/Users/alfer/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/libcairo.dylib' (no such file), '/Users/alfer/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libcairo.dylib' (no such file), 'libcairo.dylib' (no such file), '/usr/local/lib/libcairo.dylib' (no such file), '/usr/lib/libcairo.dylib' (no such file), '/Users/alfer/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/libcairo.dylib' (no such file), '/libcairo.dylib' (no such file), '/Users/alfer/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libcairo.dylib' (no such file)`

**Comment by DiegoCatalano** [comment]
The game has gone completely dark, it doesn't show the minimap anymore.
I tried to switch to OpenGL, Vulkan and it still didn't go back to normal
@WillianJunior I'm using Ubuntu too

![black_screen](https://user-images.githubusercontent.com/5301914/188030267-16a3c3cb-90c7-41cf-9d75-bb649605f8b7.png)

**Comment by MFCharles91** [comment]
+1

**Comment by alfer-aws** [comment]
at least you see something...i cant even open the game. Any workaround ?

**Comment by arthurbotelho** [comment]
> 
I just think we gotta wait valve to fix it! Sad

**Comment by israx** [comment]
+1

**Comment by tomlawrence** [comment]
+1

**Comment by maicojay16** [comment]
Same issue here, Im using MacBook Pro 2018.

![Screen Shot 2022-09-02 at 9 42 05 AM](https://user-images.githubusercontent.com/23121374/188043029-8033a453-5050-45de-a3e9-2f89097618ed.png)

Before the update everything works fine, I even play 1 game with my friend. After i update the new patch for Battlepass 2022, Im stuck in the opening loading screen of Dota.

I tried to reinstall it but nothing works. Please help!!!

**Comment by drusepth** [comment]
Looks like it crashes from the Steam Deck, too. :+1:

**Comment by nich961996** [comment]
Same bro

**Comment by abdullahkhalids** [comment]
> The game has gone completely dark, it doesn't show the minimap anymore. I tried to switch to OpenGL, Vulkan and it still didn't go back to normal @WillianJunior I'm using Ubuntu too
> 
> ![black_screen](https://user-images.githubusercontent.com/5301914/188030267-16a3c3cb-90c7-41cf-9d75-bb649605f8b7.png)

I am getting the same issue. Here is my dump [Steam System Information](https://gist.github.com/abdullahkhalids/016402dff664dcd97657b6f654fff882)

**Comment by ronakparhi** [comment]
same issue here :(

**Comment by Manthravadi** [comment]
Same issue with MacBook Pro 16-inch, 2019 model

Processor: 2.3GHz 8 core Intel Core i9
Memory: 16GB 2667 MHz DDR4
Graphics: AMD Radeon Pro 5500M 4GB + Intel UHD Graphics 630 1536 MB

Tried -vulkan and loading screen is blank(black with busy cursor).
Tried -gl and game had DOTA 2 loading screen once and blank screen twice with busy cursor always.

Had to force quit all the time. It was working fine the other night but latest BP update causes the issue.

**Comment by tuoyl** [comment]
same here

**Comment by ikasymov** [comment]
same here

**Comment by tonine1337** [comment]
Same issue. App goes to full screen and only shows a black screen. 

Current spec

![Screenshot 2022-09-02 at 4 47 55 PM](https://user-images.githubusercontent.com/14199391/188128415-d6cb1e14-7bff-4411-8ae3-7d4038b90328.png)

**Comment by FanFanFantastic** [comment]
HELP!!!

**Comment by rab343** [comment]
+1
App only shows black screen

**Comment by danginsburg** [comment]
> > The game has gone completely dark, it doesn't show the minimap anymore. I tried to switch to OpenGL, Vulkan and it still didn't go back to normal @WillianJunior I'm using Ubuntu too

This is a problem specific to OpenGL (Fog of War is not rendering correctly).  I have a fix for this that will be in the next update we ship.  That being said, be forewarned we have previously announced we plan to deprecate OpenGL and I expect we will stop supporting it in the next major update.

Regarding the main bug here (black screen on macOS): so far I've had difficulty reproducing it on a couple of Macs I have here.    I'm not yet sure what the common factor is for people experiencing it.  Can folks make sure they have the latest Steam Client (check for updates in Steam)?  The point at which people are showing their logs is just before we initalize with Steam so I wonder if it's hanging there.

**Comment by d3alexa** [comment]
macOS user here. 

Am using an extended monitor as the main display. Dota goes black screen upon loading. Switched to mirroring and Dota was able to load up but hangs with rainbow cursor whenever I try to load a custom game (not sure if normal game works)

**Comment by dota-dj** [comment]
Same thing happening here. Freezes every time as soon as the main DotA logo appears on screen. 

<img width="328" alt="Screen Shot 2022-09-02 at 4 18 27 PM" src="https://user-images.githubusercontent.com/112714488/188181448-81f72de0-e538-4b38-aa4b-b0848a47b2e8.png">

**Comment by danginsburg** [comment]
For those having the problem: does running with -nosteam on the command line make any difference?  Having difficulty repro'ing here, trying to narrow it down.

**Comment by RomanCoding** [comment]
> For those having the problem: does running with -nosteam on the command line make any difference? Having difficulty repro'ing here, trying to narrow it down.

tried with nosteam - same issue
<img width="593" alt="image" src="https://user-images.githubusercontent.com/25977049/188182496-7fbf5046-c5ba-4412-82d1-e6559b8b824b.png">

**Comment by dota-dj** [comment]
> For those having the problem: does running with -nosteam on the command line make any difference? Having difficulty repro'ing here, trying to narrow it down.

Same, also tried -nosteam with no difference.

**Comment by shmykyta** [comment]
@danginsburg tried all flags what fixed issues before, but no luck here. 
Me and my friends with MacBooks pro can't launch dota, just black window, with -gl flag we spot dota logo, but nothing further.

**Comment by omarsamad-plutoflume** [comment]
+1 ,  Ubuntu 20.04.2 LTS

**Comment by jeromeza** [comment]
+1 same issue...

macOS Monterey 12.4

**Comment by sergsibe** [comment]
+1
macOS Monterey 12.2

**Comment by omarsamad-plutoflume** [comment]
+1 Ubuntu 20.04.2

**Comment by mario-amazing** [comment]
+1 
macOS Big Sur 11.6.8

**Comment by Perc-on1** [comment]
<img width="572" alt="Screen Shot 2565-09-02 at 23 37 02" src="https://user-images.githubusercontent.com/70796253/188199322-246fa5ba-6b53-424a-ae45-b60fd983555f.png">

After launching game - Black window and keeps freezing.

**Comment by ntphong90** [comment]
Stuck on blackscreen after open. 
`failed to dlopen "/Users/phongnguyen/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libcairo.dylib" error=dlopen(/Users/phongnguyen/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libcairo.dylib, 0x0009): tried: '/Users/phongnguyen/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/libcairo.dylib' (no such file), '/libcairo.dylib' (no such file), '/Users/phongnguyen/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libcairo.dylib' (no such file)
Loaded libcairo.dylib, got 0x34c62aa0`

**Comment by souljorje** [comment]
Same issue.

```
Computer Information:
    Manufacturer:  Apple
    Model:  MacBookPro11,5
    Form Factor: Laptop
    No Touch Input Detected

Processor Information:
    CPU Vendor:  GenuineIntel
    CPU Brand:  Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
    CPU Family:  0x6
    CPU Model:  0x46
    CPU Stepping:  0x1
    CPU Type:  0x0
    Speed:  2500 Mhz
    8 logical processors
    4 physical processors
    HyperThreading:  Supported
    FCMOV:  Unsupported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Supported
    AES:  Supported
    AVX:  Supported
    AVX2:  Supported
    AVX512F:  Unsupported
    AVX512PF:  Unsupported
    AVX512ER:  Unsupported
    AVX512CD:  Unsupported
    AVX512VNNI:  Unsupported
    SHA:  Unsupported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Unsupported

Operating System Version:
    MacOS 10.14.6 (64 bit)

Video Card:
    Driver:  AMD Radeon R9 M370X
    Driver Version Not Detected
    OpenGL Version: 4.1
    Desktop Color Depth: 32 bits per pixel
    Monitor Refresh Rate: 0 Hz
    VendorID:  0x1002
    DeviceID:  0x6821
    Revision Not Detected
    Number of Monitors:  1
    Number of Logical Video Cards:  2
    Primary Display Resolution:  1440 x 900
    Desktop Resolution: 1440 x 900
    Primary Display Size: 13.03" x 8.11" (15.31" diag)
                                            33.1cm x 20.6cm (38.9cm diag)
    Primary VRAM: 2048 MB

Memory:
    RAM:  16384 MB

VR Hardware:
    VR Headset: None detected

Miscellaneous:
    UI Language:  English
    Media Type:  DVD
    Total Hard Disk Space Available:  476902 MB
    Largest Free Hard Disk Block:  20217 MB

Storage:
    Number of SSDs: 0
    Number of HDDs: 0

```

Logs for ` ~/Library/Application\ Support/Steam/steamapps/common/dota\ 2\ beta/game/dota.sh`:

```
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libengine2.dylib, got 0x7fa974d0c290
Using breakpad crash handler
[S_API] SteamAPI_Init(): Loaded '/Users/jorje/Library/Application Support/Steam/Steam.AppBundle/Steam/Contents/MacOS/steamclient.dylib' OK.
Setting breakpad minidump AppID = 570
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
found breakpad via in process memory: '/Users/jorje/Library/Application Support/Steam/Steam.AppBundle/Steam/Contents/MacOS/Frameworks/Breakpad.framework/Versions/A/Resources'
Installing breakpad crash handler
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
SteamID:  0, universe Public
SteamInternal_SetMinidumpSteamID:  Caching Steam ID:  76561198183972289 [API loaded yes]
SteamInternal_SetMinidumpSteamID:  Setting Steam ID:  76561198183972289
SteamID:  76561198183972289, universe Public
SteamID:  0, universe Public
Setting breakpad minidump AppID = 373300
Loaded libSDL2-2.0.0.dylib, got 0x7fa977c11c90
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libtier0.dylib, got 0x7fa974c05e50
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libfilesystem_stdio.dylib, got 0x7fa977c38d20
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libinputsystem.dylib, got 0x7fa977b3b6c0
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/liblocalize.dylib, got 0x7fa977b3bbd0
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/librendersystemgl.dylib, got 0x7fa977b4dc60
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/librendersystemvulkan.dylib, got 0x7fa977b4d6d0
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libresourcesystem.dylib, got 0x7fa977b559a0
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libschemasystem.dylib, got 0x7fa977b560a0
Attempting to create OpenGL context: v3.3 (Core profile)...success. Checking entry points...success.
[mvk-info] MoltenVK version 1.1.0. Vulkan version 1.1.154.
	The following 62 Vulkan extensions are supported:
		VK_KHR_16bit_storage v1
		VK_KHR_8bit_storage v1
		VK_KHR_bind_memory2 v1
		VK_KHR_create_renderpass2 v1
		VK_KHR_dedicated_allocation v3
		VK_KHR_depth_stencil_resolve v1
		VK_KHR_descriptor_update_template v1
		VK_KHR_device_group v4
		VK_KHR_device_group_creation v1
		VK_KHR_driver_properties v1
		VK_KHR_external_fence v1
		VK_KHR_external_fence_capabilities v1
		VK_KHR_external_memory v1
		VK_KHR_external_memory_capabilities v1
		VK_KHR_external_semaphore v1
		VK_KHR_external_semaphore_capabilities v1
		VK_KHR_get_memory_requirements2 v1
		VK_KHR_get_physical_device_properties2 v2
		VK_KHR_get_surface_capabilities2 v1
		VK_KHR_image_format_list v1
		VK_KHR_maintenance1 v2
		VK_KHR_maintenance2 v1
		VK_KHR_maintenance3 v1
		VK_KHR_multiview v1
		VK_KHR_portability_subset v1
		VK_KHR_push_descriptor v2
		VK_KHR_relaxed_block_layout v1
		VK_KHR_sampler_mirror_clamp_to_edge v3
		VK_KHR_sampler_ycbcr_conversion v14
		VK_KHR_shader_draw_parameters v1
		VK_KHR_shader_float16_int8 v1
		VK_KHR_shader_subgroup_extended_types v1
		VK_KHR_storage_buffer_storage_class v1
		VK_KHR_surface v25
		VK_KHR_swapchain v70
		VK_KHR_swapchain_mutable_format v1
		VK_KHR_uniform_buffer_standard_layout v1
		VK_KHR_variable_pointers v1
		VK_EXT_debug_marker v4
		VK_EXT_debug_report v9
		VK_EXT_debug_utils v2
		VK_EXT_fragment_shader_interlock v1
		VK_EXT_host_query_reset v1
		VK_EXT_image_robustness v1
		VK_EXT_inline_uniform_block v1
		VK_EXT_memory_budget v1
		VK_EXT_metal_surface v1
		VK_EXT_robustness2 v1
		VK_EXT_scalar_block_layout v1
		VK_EXT_shader_stencil_export v1
		VK_EXT_shader_viewport_index_layer v1
		VK_EXT_swapchain_colorspace v4
		VK_EXT_texel_buffer_alignment v1
		VK_EXT_vertex_attribute_divisor v3
		VK_MVK_macos_surface v3
		VK_MVK_moltenvk v28
		VK_AMD_gpu_shader_half_float v2
		VK_AMD_negative_viewport_height v1
		VK_AMD_shader_trinary_minmax v1
		VK_INTEL_shader_integer_functions2 v1
		VK_GOOGLE_display_timing v1
		VK_NV_glsl_shader v1
[mvk-info] GPU device:
		model: AMD Radeon R9 M370X
		type: Discrete
		vendorID: 0x1002
		deviceID: 0x6821
		pipelineCacheUUID: 00002774-0000-2715-5D1E-B17068167C85
	supports the following Metal Versions, GPU's and Feature Sets:
		Metal Shading Language 2.1
		macOS GPU Family 2 v1
		macOS GPU Family 1 v4
		macOS GPU Family 1 v3
		macOS GPU Family 1 v2
		macOS GPU Family 1 v1
		macOS Read-Write Texture Tier 2
[mvk-info] GPU device:
		model: Intel Iris Pro Graphics
		type: Integrated
		vendorID: 0x8086
		deviceID: 0x0d26
		pipelineCacheUUID: 00002774-0000-2714-5D1E-B17068167C85
	supports the following Metal Versions, GPU's and Feature Sets:
		Metal Shading Language 2.1
		macOS GPU Family 1 v4
		macOS GPU Family 1 v3
		macOS GPU Family 1 v2
		macOS GPU Family 1 v1
[mvk-info] Created VkInstance with the following 4 Vulkan extensions enabled:
		VK_KHR_get_physical_device_properties2 v2
		VK_KHR_surface v25
		VK_MVK_macos_surface v3
		VK_MVK_moltenvk v28
[mvk-info] Using MTLFence for Vulkan semaphores.
[mvk-info] Created VkDevice to run on GPU AMD Radeon R9 M370X with the following 9 Vulkan extensions enabled:
		VK_KHR_create_renderpass2 v1
		VK_KHR_depth_stencil_resolve v1
		VK_KHR_descriptor_update_template v1
		VK_KHR_image_format_list v1
		VK_KHR_maintenance1 v2
		VK_KHR_maintenance2 v1
		VK_KHR_swapchain v70
		VK_KHR_swapchain_mutable_format v1
		VK_EXT_memory_budget v1
[mvk-info] Created 3 swapchain images with initial size (1440, 900).
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libmaterialsystem2.dylib, got 0x7fa95065b790
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libmeshsystem.dylib, got 0x7fa953022040
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libworldrenderer.dylib, got 0x7fa953085870
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libvscript.dylib, got 0x7fa9502eeb50
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libnetworksystem.dylib, got 0x7fa950643c50
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libanimationsystem.dylib, got 0x7fa9530ad3a0
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libvphysics2.dylib, got 0x7fa950787ce0
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libsoundsystem.dylib, got 0x7fa9507884f0
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libscenesystem.dylib, got 0x7fa9507b15b0
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libparticles.dylib, got 0x7fa9530bb840
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/dota/bin/osx64/libserver.dylib, got 0x7fa953428ef0
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/dota/bin/osx64/libclient.dylib, got 0x7fa9507b2ef0
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libpanorama.dylib, got 0x7fa95360f040
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libpanorama_text_pango.dylib, got 0x7fa95320e310
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libv8system.dylib, got 0x7fa953732280
Loaded libmpg123.dylib, got 0x7fa9533fb130
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libfreetype.dylib, got 0x7fa95320dec0
Fontconfig warning: "/Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/core/panorama/fonts/conf.d/41-repl-os-win.conf", line 148: Having multiple values in <test> isn't supported and may not work as expected
Fontconfig warning: "/Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/core/panorama/fonts/conf.d/41-repl-os-win.conf", line 160: Having multiple values in <test> isn't supported and may not work as expected
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libcairo.dylib, got 0x0
 failed to dlopen "/Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libcairo.dylib" error=dlopen(/Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libcairo.dylib, 9): image not found
Loaded libcairo.dylib, got 0x0
 failed to dlopen "libcairo.dylib" error=dlopen(libcairo.dylib, 9): image not found
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libengine2.dylib, got 0x7fa974d0c290
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/dota/bin/osx64/libhost.dylib, got 0x7fa953614c90
Loaded /Users/jorje/Library/Application Support/Steam/steamapps/common/dota 2 beta/game/bin/osx64/dota2.app/Contents/MacOS/libscenefilecache.dylib, got 0x7fa953616490
```

**Comment by shrakuma** [comment]
+1 Same issue here on mac . Monterey 12.4 16 gig ram Intel graphics card iris 1536 mb 
Hangs when opened and gets stuck.

**Comment by alfer-aws** [comment]
With -nosteam no change, still black screen and app not responding.
Steam client is also up to date @danginsburg 

<img width="578" alt="image" src="https://user-images.githubusercontent.com/92041712/188213325-a0a0acf7-426b-4a01-9af5-0fabe0a71ce8.png">

OS also up to date

**Comment by zxcdet** [comment]
Gaben fix it

**Comment by mikitahimpel** [comment]
+1

**Comment by petrov-pavel** [comment]
same here

**Comment by Inder-Dhillon** [comment]
+1

**Comment by mikeavvad** [comment]
https://www.volvocars.com/ call volvo, the time is now

**Comment by ChaitanyaKhoje** [comment]
+1 same thing happening on my MB Pro 2017

**Comment by viciecal** [comment]
> For those having the problem: does running with -nosteam on the command line make any difference? Having difficulty repro'ing here, trying to narrow it down.
@danginsburg 

Hello friend, some info about this bug. Using latest steam version, latest driver, etc.

-novid, -nosteam, -console, etc that type of commands are not making anything different. bug behaviour still the same
if you choose -vulkan, game launches and it's straight up black screen forever while it's "loading"
if you choose -gl, game launches and shows dota icon forever while it's "loading"
if you choose -safe, first scenario happens (same as choosing -vulkan)
I'm also using MacBook pro 2018, intel based processor. Was running fine jsut before the update.
tried playing around values within video.txt but it's not helping.
may it be related to intel processors? people here is using both monterrey and big sur, with the same issue using 2017-2020 macs.

**Comment by Zoktog** [comment]
More than 20 hours have already passed since the players reported the problem, but no one from the company contacted the players, did not fix it, did not admit the errors and did not inform about the timing of when they will fix it.

**Comment by ChaitanyaKhoje** [comment]
I was able to get into the game and buy the battle pass, but when I opened it, the game froze, had to force quit, and now back to the black screen, game won't start.

**Comment by viciecal** [comment]
> More than 20 hours have already passed since the players reported the problem, but no one from the company contacted the players, did not fix it, did not admit the errors and did not inform about the timing of when they will fix it.

Sorry this is not a forum. This comment is Not helpful regarding the main issue. This is a bug tracker. Those type of complaints are more appropiate for other website.

**Comment by mallikarjun-b-r** [comment]
A billion dollar company, doesn't test its product before releasing what a joke

**Comment by viciecal** [comment]
Please stop using this bug tracker as forum. Stop ruining the integrity of github don't boycott it please. This is NOT the right place for that.
Look at the first comments , this is what you post here. Logs, system info, some type of help with reproducing the bug, etc. 
Thanks

**Comment by viciecal** [comment]
@danginsburg Apparently a bunch of people from the forum is confirming that this is happening only on intel macs. M1 macs can at least start the game.

**Comment by tt3kno** [comment]
<img width="344" alt="Bildschirmfoto 2022-09-02 um 22 31 49" src="https://user-images.githubusercontent.com/112729637/188231434-77b75f2d-f5b5-435d-b02b-98f6b6be6cd7.png">

<img width="1434" alt="Bildschirmfoto 2022-09-02 um 22 33 52" src="https://user-images.githubusercontent.com/112729637/188231695-5778ae9f-e3a3-4c55-b78f-3793687ebe38.png">

After pressing play I am stuck in the loading screen with the Dota 2 Logo showing.

**Comment by bvndicut3** [comment]
same here no dota since update

**Comment by dkalantari** [comment]
> Please stop using this bug tracker as forum. Stop ruining the integrity of github don't boycott it please. This is NOT the right place for that. Look at the first comments , this is what you post here. Logs, system info, some type of help with reproducing the bug, etc. Thanks

Fair point and technically correct, but if you search Valve for any type of customer support forum you will get no where but land here in their github page. Still agree with you but how else can customers voice a systematic issue? also note there has been no single announcement by the company so it falls on Valve to monitor this space.

**Comment by aungus** [comment]
Not sure if the 100MB update was recent but problem persists.

**Comment by tt3kno** [comment]
There was an update a couple of minutes ago. Didn't fix the problem for me neither though.

**Comment by fveracoechea** [comment]
That's right, just downloaded a small update but it didn't fix the problem at all...

![Screen Shot 2022-09-01 at 9 22 36 PM](https://user-images.githubusercontent.com/26258895/188238483-acb04da4-5080-440a-a5f7-de734807133c.png)

It looks like our three-day weekend will be spent waiting on a fix...

**Comment by reutsharabani** [comment]
> > > The game has gone completely dark, it doesn't show the minimap anymore. I tried to switch to OpenGL, Vulkan and it still didn't go back to normal @WillianJunior I'm using Ubuntu too
> 
> This is a problem specific to OpenGL (Fog of War is not rendering correctly). I have a fix for this that will be in the next update we ship. That being said, be forewarned we have previously announced we plan to deprecate OpenGL and I expect we will stop supporting it in the next major update.
> 
> Regarding the main bug here (black screen on macOS): so far I've had difficulty reproducing it on a couple of Macs I have here. I'm not yet sure what the common factor is for people experiencing it. Can folks make sure they have the latest Steam Client (check for updates in Steam)? The point at which people are showing their logs is just before we initalize with Steam so I wonder if it's hanging there.

The common factor of machines that hang is that they are **intel** machines.
I assume you're using M1s/M2s to test.

How was this not tested before BP release? It fails for me and for anyone with mac I know (and I know many because I work in tech).

It also fails on imacs not just mbps.

If dota is no longer supported on mac (obviously it isn't being tested on mac) it should be noted. I ended up installing windows through bootcamp because I'd like to play while battlepass is still available. Works fine on bootcamp obviously.

This is my suggestion to anyone here: Install windows through bootcamp. This wasn't properly tested during BP development so it may take time to fix. I think you can use windows trial until valve fix it so you don't even have to buy windows.

**Comment by MohmmadSawalha-ios** [comment]
Same issue ..

**Comment by smh-freelancer** [comment]
Same issue.
hackintosh monterey.

Startup options :
-vulkan -vconsole => not start & show Nothing (black with busy cursor)
Had to force quit.

Startup options :
-gl => not start & show dota2 logo (logo with busy cursor)
Had to force quit.


last week whit hackintosh big sur everything was working fine.

**Comment by danginsburg** [comment]
We finally got this figured out.  It was a tricky problem due to a change in our low level memory allocator and it didn't repro on most of our machines.  Fix is checked in, it will be in the next update.

**Comment by Rasoulsa** [comment]
good job, can you tell us about the released time for update ?!

**Comment by fveracoechea** [comment]
@danginsburg Awesome Thanks! 🎉

**Comment by danginsburg** [comment]
> good job, can you tell us about the released time for update ?!

As soon as the bits clear our build machine :)

**Comment by wassgang93** [comment]
> > good job, can you tell us about the released time for update ?!
> 
> As soon as the bits clear our build machine :)

So like within an hour?

**Comment by viciecal** [comment]
Yeahhh finally. Let's go. Thanks.

**Comment by P4SH4P** [comment]
So did it work out? 🧐

**Comment by 13adada** [comment]
anybody know when the update will be?

**Comment by fveracoechea** [comment]
@P4SH4P it looks like the fix has not been applied yet

**Comment by akashsinghkunwar** [comment]
Its fixed now. You can play in MAC now. Thanks

**Comment by israx** [comment]
Thanks. It's working now

**Comment by viciecal** [comment]
just confirming it's working for me on a 2018 mac with intel - just fine as before the update

**Comment by tuoyl** [comment]
Guys, the new released updates fix this problem.

**Comment by tonine1337** [comment]
Working on macbook pro 2017, while connected to the external display. And playing the game on that.

**Comment by tuoyl** [comment]
> We finally got this figured out. It was a tricky problem due to a change in our low level memory allocator and it didn't repro on most of our machines. Fix is checked in, it will be in the next update.

Awesome, thanks for your hard work!

**Comment by bhartimaeus** [comment]
Issue isn't solved for me. Macbook 2020 Intel chip. The game does start, but runs at 10ish fps and is unplayable. Had a game crash as well. Thanks for working on it, but please resolve this. 

If I'm doing something wrong please let me know.

**Comment by MilanOnTheMoon** [comment]
Still not resolved for my mac either. 2018 Intel MB Pro. Game starts then at 1:20 every time it red screens and crashes me out. Have logs if devs need. Never had this issue before the BP update so not sure what could be causing it. Messed with some settings in launch options and in game config as well as autoconfig. Verified and reinstalled fresh. I'm out of ideas

**Comment by kisak-valve** [comment]
Hello @bhartimaeus, @ImAMilano, the issue tracked here is a failure to reach the main menu during initialization. Please open a new issue report so that your issues can be tracked properly.
